src/VehicleSelector/VehicleSelector.types.ts

```diff
@@ -39,7 +39,7 @@ export type VehicleSelectorProps = {
   /**
    * Allow multiple selections
    */
-  multipleSelection?: boolean;
```

Comment:
This is a breaking change. Did you find a specific case where you needed to support multipleGates = true and multipleVariants = false?

Reply:
Ideally we would avoid a breaking change as you'll need to go and update everywhere that this is used, and target v9 instead.

If we do need to support variable combinations of `multiple` then I'd prefer that we did it in a nested object:

```ts
type Props = {
  /**
   * Allow multiple selections
   */
  multiple?: {
     /**
      * For the variant field
      */
     variant?: boolean;
     /**
      * For the gate field (if present)
      */
     gate?: boolean;
  }
}
```

Reply:
After discussing with Dave and Sowbhagya, decided to have only one prop multipleSelection for now.